src/chlorine/ui/connection.cljs

```diff
@@ -46,9 +46,15 @@
 (defn conn-view [cmd]
   (let [div (. js/document (createElement "div"))
         panel (.. js/atom -workspace (addModalPanel #js {:item div}))
-        port-file (-> js/atom .-project .getPaths first
-                      (str "/.shadow-cljs/socket-repl.port"))]
-    (when (existsSync port-file)
+        port-file (->> ["/.shadow-cljs/socket-repl.port"
+                        "/.socket-repl-port"
+                        "/.nrepl-port"]
+                       (map (fn [path]
+                             (-> js/atom .-project .getPaths first
+                                 (str path))))
```

Comment:
This should not be done - it'll not work on Windows. Can you change to `(.join path ...)` instead, of the "path" library on Node?